vuepress/docs/next/docs/compose/ecm-overview.md

```diff
@@ -0,0 +1,20 @@
+# Entando Component Manager
+
+​​An Entando Application is composed of the Entando App Builder, Entando App Engine, and Entando Component Manager. The Entando Component Manager (ECM) is a service that links the [Entando Component Repository](ecr-overview.md) (ECR) of the App Builder to the core application instance. It appears as `quickstart-cm-deployment` in the Kubernetes pod list:
+
+![pods.png](./img/pods.png)
+
+The ECM communicates with the Entando Kubernetes service to populate the ECR with the bundles deployed as [Custom Resources](../consume/custom-resources.md) in the Entando cluster. These bundles are available to the application and can be managed from within the ECR. 
+
+<img src="./img/ecm-flow.png" width="800" height="597.73">
```

Comment:
If we're going to include the numbers, maybe we should explain the steps. 1) A user goes to the ECR in the browser, 2) a request is made to the digital-exchange endpoint, 3) which is served by the Component Manager, 4) which calls the k8s-service to return the list of available bundles in the namespace.  A similar flow occurs when bundles are installed or uninstalled, with additional work done by the Operator which watches for events on the appropriate resources.